lab-judy/server.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+// npm modules
+const morgan = require('morgan');
+const express = require('express');
+const debug = require('debug')('note:server');
+
+// app modules
+const personRouter = require('./route/person-router.js');
+
+//module constants
+const PORT = process.env.PORT || 3000;
+const app = express();
+
+app.use(morgan('dev'));
+
+//register routes
+app.use(personRouter);
+
+
+
+
```

Comment:
you are currently not making use of your error-handling middleware. You'll need an app.use where you pass in your error-handling middleware after your router.